### PR TITLE
Update the normative reference from Bluetooth 4.1 to 4.2.

### PR DESCRIPTION
**Pull request by jyasskin**
Demo at https://rawgit.com/jyasskin/web-bluetooth-1/update-bt-standard/index.html.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2725,7 +2732,15 @@
               <ol type="A">
                 <li value="2">Service Discovery Protocol (SDP) Specification
                   <ol>
-                    <li value="5"><dfn>Searching for Services</dfn></li>
+                    <li value="2">Overview
+                      <ol>
+                        <li value="5"><dfn>Searching for Services</dfn>
+                          <ol>
+                            <li value="1"><dfn>UUID alias</dfn></li>
```
Comment (scheib) [inline index.html]: As you do elsewhere, perhaps use name of section and parenthetically refer to definition: UUID (defines < dfn>UUID alias</dfn>)

File: index.html [hunk 2]
```diff
@@ -2292,8 +2294,8 @@
         <dd></dd>
         <dt>UUID canonicalUUID(unsigned long alias)</dt>
         <dd>
-          Returns a 128-bit <a>UUID</a> given a 16- or 32-bit Bluetooth <a title="UUID aliases">UUID alias</a>.
-          The algorithm for converting an alias to a full 128-bit UUID is defined in [[!BLUETOOTH41]] 3.F.3.2.1:
+          Returns a 128-bit <a>UUID</a> given a 16- or 32-bit Bluetooth <a>UUID alias</a>.
+          The algorithm for converting an alias to a full 128-bit UUID is defined in [[!BLUETOOTH42]] 3.B.2.5.1:
```
Comment (scheib) [inline index.html]: Perhaps name the reference instead of just the section reference? e.g. "UUID alias 128_bit_value", "UUID alias formula"?

**Comment by scheib** [comment]
LGTM. Comments optional on removing number based sub-section references, and sub section naming.

File: index.html (hunk 1 above)
Comment (jyasskin) [inline index.html]: Sure. Btw, you can make tags render verbatim by surrounding them with backticks (`).

File: index.html (hunk 2 above)
Comment (jyasskin) [inline index.html]: 'k. Done as:

> The UA MUST return `<a>`the 128-bit UUID represented`</a>` by the 16- or 32-bit UUID alias _alias_.
> `<p class="note">`
> This algorithm consists of replacing the top 32 bits of "`00000000-0000-1000-8000-00805f9b34fb`" with the bits of the alias. For example, `canonicalUUID(0xDEADBEEF)` returns `"deadbeef-0000-1000-8000-00805f9b34fb"`.
> `</p>`

**Pull request merged**